lib/ansible/cli/doc.py

```diff
@@ -508,6 +500,11 @@ def get_man_text(self, doc):
                 text.append('%s: %s' % (k.upper(), textwrap.fill(CLI.tty_ify(doc[k]), limit - (len(k) + 2), subsequent_indent=opt_indent)))
             elif isinstance(doc[k], (list, tuple)):
                 text.append('%s: %s' % (k.upper(), ', '.join(doc[k])))
+            elif k == 'metadata':
+                metadata_block = self.get_metadata_block(doc)
```

Comment:
Does this throw any exception? In the code you're removing it's inside `try:`.

Reply:
No. I think this is necessary with new code.